Rakefile

```diff
@@ -7,5 +7,6 @@ end
 
 desc 'runs game'
 task :play do
+  trap('INT') { exit }
```

Comment:
This in two places. Is there a way we can remove the duplication?

Reply:
Figured out that I can avoid a rake aborted message by rescuing signal exceptions and doing nothing.  Have replaced this line with such a rescue.